### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: src/hydra/fetchJsonLd.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
  * @param {object} options
  * @return {Promise.<object>} An object with a response key (the original HTTP response) and an optional body key (the parsed JSON-LD body)
  */
-export default (url, options = {})  => {
+export default function(url, options = {}) {
```
Comment (dunglas) [inline src/hydra/fetchJsonLd.js]: Why this change?

Comment (dunglas) [inline src/hydra/fetchJsonLd.js]: Ok got it thanks to @Gregcop1 👍

**Comment by dunglas** [comment]
Thanks! Can you just fix ESLint reports?

**Comment by soyuka** [comment]
Done sorry, I'm using https://github.com/feross/standard/ usually and therefore don't use any semicolons.

File: src/hydra/fetchJsonLd.js (hunk 1 above)
Comment (soyuka) [inline src/hydra/fetchJsonLd.js]: Added a name also, it's better to avoid anonymous functions which can be a pain to debug.

File: src/hydra/parseHydraDocumentation.js [hunk 2]
```diff
@@ -30,9 +32,9 @@ const findSupportedClass = (docs, supportedClass) => {
   }
 
   throw new Error(`The class ${supportedClass} doesn't exist.`);
-};
+}
 
-export const getDocumentationUrlFromHeaders = (headers) => {
+function getDocumentationUrlFromHeaders(headers) {
```
Comment (soyuka) [inline src/hydra/parseHydraDocumentation.js]: I think that this export wasn't used. Lmk if it should still be exported

Comment (dunglas) [inline src/hydra/parseHydraDocumentation.js]: It is used in another library. Please keep it exported.

Comment (soyuka) [inline src/hydra/parseHydraDocumentation.js]: Hmm, then you have a named export and a default one, feels wrong :p. 

Fixed.

**Comment by dunglas** [comment]
Thank you @soyuka!
